src/core/dbus.c

```diff
@@ -854,6 +860,10 @@ int bus_init_api(Manager *m) {
         if (r < 0)
                 return log_error_errno(r, "Failed to set up API bus: %m");
 
+        r = bus_track_coldplug(bus, &m->subscribed, false, m->deserialized_subscribed);
```

Comment:
our current coding style suggests to put parameter comments before opaque parameters, i.e. the third param here should be prefied with `/* recursive= */`